server/events/models/models.go

```diff
@@ -505,6 +536,10 @@ func (p ProjectPlanStatus) String() string {
 		return "applied"
 	case DiscardedPlanStatus:
 		return "plan_discarded"
+	case ErroredPolicyCheckStatus:
+		return "policy_check_errored"
+	case PassedPolicyCheckStatus:
```

Comment:
Should we differentiate between Passed and Overriden? I would imagine so.

Reply:
We can add that when we start working on that logic. But I agree we should have another status for that.